cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -79,6 +79,29 @@ get_home() {
         getent passwd "$(id -un)" | cut -d: -f6
     fi
 }
+
+# Check if running on Windows with POSIX-compliant shell (CYGWIN, MSYS, MINGW)
+is_windows_posix() {
+    case "$(uname)" in
+        CYGWIN*|MSYS*|MINGW*)
+            return 0
+            ;;
+        *)
+            return 1
+            ;;
+    esac
+}
+# Convert POSIX path to Windows path
+convert_path_for_receipt() {
+    local _path="$1"
+    if is_windows_posix && command -v cygpath > /dev/null 2>&1; then
+        # Use cygpath to convert path, then escape backslashes for JSON
+        cygpath -w "$_path" | sed 's/\\/\\\\/g'
+    else
+        echo "$_path"
```

Comment:
In this case we know the path is likely to be wrong but we're not going to be able to convert it, right?